README.md

```diff
@@ -1,2 +1,17 @@
 # sourcerer-app
 Sourcerer app makes engineering profiles from their git repositories.
```

Comment:
I don't argue about the wording. Just some people prefer to have separate issue for not related changes. It's more about style, which differs from company to company, from team to team, etc